images/origin/Dockerfile

```diff
@@ -16,5 +16,6 @@ RUN ln -s /usr/bin/openshift /usr/bin/osc && \
     ln -s /usr/bin/openshift /usr/bin/openshift-router
 
 ENV HOME /root
+ENV KUBECONFIG /var/lib/openshift/openshift.local.certificates/admin/.kubeconfig
```

Comment:
can't you set multiple ENV vars? if so, you can move this up and save one layer ;-)